back-end/student-tracker-template.yaml

```diff
@@ -39,6 +39,33 @@ Resources:
       Role: !GetAtt IAMRole.Arn
     DependsOn:
       - DynamoDBTable
+  CoursesFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        ZipFile: |
+          const AWS = require('aws-sdk')
+          const dbClient = new AWS.DynamoDB.DocumentClient()
+          exports.handler = async(event)=>{
+            let body;
+            let statusCode = 200;
+            try{
+              body = await dbClient.put(event).promise();
```

Comment:
This new lambda function should be to *get all of the courses from the database, not to put them in. You will need to find the code to retrieve, it should be dbclient.query or dbclient.scan